src/components/ControlButton/ControlButton.scss

```
@@ -0,0 +1,7 @@
+.dc-control-button {
+    position: relative;
+    color: var(--yc-color-text-secondary);
+    width: 28px;
+    height: 28px;
```

Давай вынесем в переменную на уровне селектора?

```suggestion
    $buttonSize: 28px;

    width: $buttonSize;
    height: $buttonSize;
```